library/config.c

```diff
@@ -24,6 +24,14 @@ limitations under the License.
 
 const char* ZITI_SDK_CONFIG = "ZITI_SDK_CONFIG";
 
+const char* APP_ID = NULL;
+const char* APP_VERSION = NULL;
+
+void ziti_set_app_info(const char *app_id, const char *app_version) {
+    APP_ID = strdup(app_id);
```

Comment:
i guess i shouldn't call this function 1000's of times right? :)

Reply:
updated, knock yourself out